canonical/store-components
# Create integration card

## Done
- Created integration card with channel data 

## QA
- Go to https://store-components-110.demos.haus/?path=/docs/integrationcard--docs
- or run `yarn docs` and go to http://localhost:9009/?path=/docs/integrationcard--docs
- Check that channel data `"latest/stable"` is in the integration cards

## Fixes
Fixes: #https://warthogs.atlassian.net/browse/WD-8386
